desc/equilibrium/equilibrium.py

```diff
@@ -427,9 +473,9 @@ def __repr__(self):
             type(self).__name__
             + " at "
             + str(hex(id(self)))
-            + " (L={}, M={}, N={}, NFP={}, sym={}, spectral_indexing={})".format(
-                self.L, self.M, self.N, self.NFP, self.sym, self.spectral_indexing
-            )
+            + f" (L={self.L}, M={self.M}, N={self.N}, NFP={self.NFP},"
+            + f" NFP_umbilic_factor={self.NFP_umbilic_factor}, sym={self.sym},"
```

Comment:
not sure if we need to include the umbilic stuff here, since this is mainly just used when printing very summarized information about the object